ecosystem/sep-0006.md

```diff
@@ -67,6 +67,50 @@ Valid CORS headers are necessary to allow web clients from other sites to use th
+
+### Basic Wallet Implementation
+
+* Identify assets you want to support manually in advance
+* For each asset, use information from its `stellar.toml` and its `/info` endpoint to display useful information to the user like transaction fee structure, types of deposits and withdrawal supported, and details about the anchor like street address.
+* Use `/info` to determine:
+    * anchor endpoints that require authentication, and perform [authentication](#authentication) via SEP-10 before hitting those endpoints
+    * Fee structure: if `fee_fixed` and `fee_percent` are provided, show this to the user early in the process so they're fully informed.
+    * If the `/fee` endpoint is enabled, use it for computing fees when you need to show them to the user.
+* While `/info` allows an anchor to communicate non-standard fields that are needed for `/deposit` or `/withdraw`, it's easier for a basic wallet implemention to hard-code extra fields that are needed on a per-anchor basis, and ensure those fields are passed in properly.
+* As a conservative measure, pass in any optional fields (including amount, which you should ask the user for) for `/deposit` or `/withdraw` that the wallet has on-hand, such as `email_address` and `account`.
+* For `/deposit`
+    * If the issuer's `/deposit` endpoint immediately returns success:
+        * display the deposit info that came back from the endpoint to the user, including fee and you're done! The user will execute the deposit exernally using the instructions if they want to.
+    * If the issuer responds with an interactive flow, handle it as [described in detail](#3-customer-information-needed-interactive). We expect this is the most common flow, and it should work smoothly.
+    * For now, do not handle the non-interactive or status responses to the `/deposit` request.
+    * Handle the [special cases](#special-cases), they're relatively common.
+* For `/withdraw`
+    * If the issuer's `/withdraw` endpoint immediately returns success:
+        * Provide an interface to the user that allows them to pick a withdrawal amount and view the total fee. The user can then confirm and your wallet will initiate the withdrawal by sending the Stellar payment to the address provided by the issuer.
+    * If the anchor responds with an interactive flow, handle it as [described in detail](#3-customer-information-needed-interactive).
+    * For now, do not handle the non-interactive or status responses to the `/withdraw` request.
+* It provides a better experience for users to show deposits or withdrawal they've completed in the past via the `/transactions` endpoint, but it's not strictly necessary.
+
+### Basic Anchor Implementation
+
+* Provide a full-featured implementation of `/info`. The more info you provide there, the better a wallet can relay deposits and withdrawals from the user to you.
```

Comment:
I think implementing the `"fields"` part of both `"deposit"` and `"withdraw"` aren't really useful if we're favoring the interactive flow.

What do you think about recommending anchors to focus on the other parts of the `/info` endpoint (enabled, fees, amounts), rather than asking them to provide as much info as possible?